<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="engBtn" >
   <property name="geometry" >
    <rect>
     <x>14</x>
     <y>30</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>English</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chsBtn" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>30</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Chinese</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loveBtn" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>100</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>I Love My Family</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testBtn" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>100</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>test tr</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
